src/content/blogs/onelab-launch-at-linaro-connect-madrid-2024.mdx

```diff
@@ -0,0 +1,54 @@
+
+In this blog, I would like to drive you through all the activities organised at Linaro Connect where participants could learn more about how ONELab is revolutionizing the way we test and make interoperable Edge and IoT devices. 
+
+We started with a panel session during the Wednesday morning keynote, led by Linaro CTO Grant Likely, who was joined by Tomas Evensen from AMD, Jonathan Humphreys from TI, Guru Chinnabhandar from Qualcomm, Peter Robinson from Red Hat and Sato-san from Renesas. 
+
+![ONELab keynote panel at Linaro Connect Madrid 2024](/linaro-website/images/blog/ONELab-keynote-panel_yenmud.png)
+*ONELab keynote panel at Linaro Connect Madrid 2024 <br />*
+
+The panel session developed into a vibrant conversation around the reasons why joining the ONELab project, focusing on the extreme interoperability features that the lab can provide to different players. The Lab indeed guarantees that applications and different OSs work seamlessly across the entire spectrum, it also guarantees SystemReady standard compliance, meaning that collaboration is enhanced without the need of custom solutions for each different enablement, with sensitive improvements on reducing operational timing, better focus and readiness.
+
+In the afternoon, we showed ONELab in action and things became more focused with a Plugfest followed by an Advisory meeting. Don Harbin and Daniel Thompson from Linaro (who are leading the partner experience and technical support), introduced the plan to release ONELab officially in Q4. 
+
+![ONELab - IoT HW ODM Onboarding](/linaro-website/images/blog/ONELab-IoT-HW-ODM-Onboarding_njhryv.png)
+
+Among the plans discussed was migrating from testing within Linaro’s Lab to the desired virtual presence of the partner devices locally in their Labs. Systems early in their life cycle are usually in short supply and expensive, whilst prone to frequent updates, allowing early participation in the lab. Leaving the devices to be tested (DUTs) with the engineers who are experts in its bring up can significantly accelerate discovering and resolving issues uncovered.
+
+![ONELab kickoff event](/linaro-website/images/blog/ONELab-kickoff-event_yh6ylr.png)
+*ONELab kickoff event <br />*
+
+A groundbreaking highlight was the ease with which diverse partner DUTs were attached to the service. Both TI and Qualcomm had brought their hardware to the event and, in just half an hour, checks were completed and ready to run. This feature represents a key change for owners of large board farms, who can confirm that onboarding is currently fraught with pitfalls, and enablement takes days to months to accomplish. Instead with ONELab time for this operation is sensitively reduced and accomplished with extreme ease.
+
+Enabling the Qualcomm RB2 | Enabling the TI SK-AM62B
+:---:|:---:
+![](/linaro-website/images/blog/enabling-qualcomm-rb2_pxtv96.jpg) | ![](/linaro-website/images/blog/enabling-ti-sk-am62b_t5ulnz.jpg)
```

Comment:
alt text shouldn't be blank unless it's a purely decorative image https://www.w3.org/WAI/tutorials/images/decision-tree/